client/components/ProductList.jsx

```diff
@@ -36,6 +40,7 @@ class ProductList extends Component {
 }
 
 function mapState({products}) {
+	console.log('PRODUCTS', products)
```

Comment:
Ill be picky and say for exercise, make sure to take out all console.log and any code that is not yet ready to be implemented.